docs/application/native/tutorials/details/sec-privileges.md

```diff
@@ -145,24 +148,27 @@ when using security-sensitive API modules in native applications:
 | `http://tizen.org/privilege/tee.client` | partner |  | The application can call security related functions running inside a Trusted Execution Environment (TEE), which ensures that sensitive data is stored, processed, and protected in an isolated, trusted environment. |
-| `http://tizen.org/privilege/telephony` | public |  | The application can retrieve telephony information, such as the network and SIM card used, the IMEI, and the statuses of calls. |
+| `http://tizen.org/privilege/telephony` | public |  | The application can retrieve telephony information, such as the network and SIM card used and the statuses of calls. |
 | `http://tizen.org/privilege/telephony.admin` | platform |  | The application can manage telephony settings, such as those for incoming and outgoing calls, forwarding and holding calls, networks, and SIM cards. |
 | `http://tizen.org/privilege/tethering.admin` | platform |  | The application can enable and disable tethering services. This may result in additional charges depending on user's payment plan. |
 | `http://tizen.org/privilege/use_ir` | public |  | The application can use the infrared transmitter. |
 | `http://tizen.org/privilege/voicecontrol.manager` | platform |  | The application can record user's voice and recognize it so that voice commands can be used to control this app. It can also give responses to user's commands. |
+| `http://tizen.org/privilege/voicecontrol.tts` | partner |  | The application can use the voice control engine to recognize users' voice commands and provide voice feedback using its own voice. |
 | `http://tizen.org/privilege/volume.set` | public |  | The application can adjust the volume for different features, such as notification alerts, ringtones, and media. |
 | `http://tizen.org/privilege/vpnservice` | public |  | The application can manage the Virtual Private Network (VPN) and change its settings. |
 | `http://tizen.org/privilege/web-history.admin` | platform | User history | The application can manage user's internet history. |
 | `http://tizen.org/privilege/widget.viewer` | public |  | The application can show widgets, and information from their associated applications, on the home screen. |
 | `http://tizen.org/privilege/wifidirect` | public |  | The application can enable and disable Wi-Fi Direct, manage Wi-Fi Direct connections, and change Wi-Fi Direct settings. |
 | `http://tizen.org/privilege/window.priority.set` | public |  | The application can appear on top of other windows and screens, including the lock screen, according to the order of priority of the windows. This may prevent user from interacting with other applications or screens until the window for this application is closed. |
+| `http://tizen.org/privilege/windowsystem.admin` | platform | The application can change the settings for services provided by the display server, such as the quick panel and softkey bar. |
```

Comment:
The last cell should be moved to right.

![image](https://user-images.githubusercontent.com/9958374/67645745-bda57700-f96d-11e9-962c-cc21325e7966.png)

Reply:
Done.